Davi_Cruz/variable.h

```diff
@@ -0,0 +1,30 @@
+#ifndef VARIABLE_H
+#define VARIABLE_H
+
+#include "std_lib_facilities.h"
+
+struct Variable
+{
+    string name;
+    double value;
+
+    Variable(string name, double val)
+        : name{name}, value{val} {}
+};
+
+struct Table_variables
+{
+    vector<Variable> variables{};
```

Comment:
Should it be public? What other data structure can you use here?

Reply:
There are methods get_value and set_value, so it can be private. I am thinking to change it in a interface, so I will can use a file store system.